runtime/development/src/lib.rs

```diff
@@ -350,13 +398,13 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type FreezeIdentifier = ();
 	type MaxFreezes = ConstU32<10>;
-	type MaxHolds = ConstU32<10>;
 	type MaxLocks = MaxLocks;
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
 	/// The overarching event type.
 	type RuntimeEvent = RuntimeEvent;
-	type RuntimeHoldReason = ();
+	type RuntimeHoldReason = RuntimeHoldReason;
+	type RuntimeFreezeReason = RuntimeFreezeReason;
```

Comment:
It seems that we do not have any way of freezing things right now. It seems like other pallets do not use freeze. The `RuntimeFreezeReason` coming from the runtime will probably be `()`.

Reply:
Should we anyway check the accounts to ensure we're no freezing

Reply:
I checked all accounts on all three chains: The `pallet_balances::Freeze` storage is empty for all of them.

Reply:
Yeah!! Thanks!